aiosmtpd/tests/test_smtp.py

```diff
@@ -434,6 +434,13 @@ def test_mail_params_bad_syntax_esmtp(self):
                 response,
                 b'Syntax: MAIL FROM: <address> [SP <mail-parameters>]')
 
+    def test_mail_fail_parse_email(self):
+        with SMTP(*self.address) as client:
+            client.helo('example.com')
+            code, response = client.docmd('MAIL FROM: <""@example.com>')
```

Comment:
Yeah, it's tricky because we don't know when it will be fixed.  Let's say it's fixed in Python 3.6.2.   Anybody running with 3.6.1 will see the test pass, but not in 3.6.2.  We could mark it with unittest's `expectedFailure`.  Maybe we should do that *and* add a mock to guarantee code coverage?